swagger.yml

```diff
@@ -1051,6 +1074,7 @@ components:
           type: number
         UnpaidDues:
           type: number
+      description: ''
```

Comment:
description: ''????

Reply:
with Stoplight it is not visible. removed manually